lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -374,3 +413,63 @@ def _discard_tunnel_flows(self, precedence:int, i_teid:int,
             flows.delete_flow(self._datapath, self.tbl_num, match,
                               priority=priority + 1)
 
+    @set_ev_cls(ofp_event.EventOFPPacketIn, MAIN_DISPATCHER)
+    def _packet_in_handler(self, ev):
+        msg = ev.msg
+        pkt = packet.Packet(msg.data)
+        pkt_ipv4 = pkt.get_protocol(ipv4.ipv4)
+
+        dst = pkt_ipv4.dst
+        # For sending notification to SMF using GRPC
+        self._send_message_interface(dst, msg.cookie)
+        # Add flow for paging with hard time.
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP, ipv4_dst=dst)
+
+        flows.add_drop_flow(self._datapath, self.tbl_num, match, [],
+                       priority=flows.PAGING_PRIORITY + 1,
```

Comment:
Can we also define this priority(not liking the +1 here)?

Reply:
Defined the priority in utils.